azure-pipelines.yml

```diff
@@ -0,0 +1,111 @@
+parameters:
+- name: force_artifacts
+  displayName: Force Artifacts
+  type: boolean
+  default: true
+- name: skip_assembly_signing
+  displayName: Skip Assembly Signing
+  type: boolean
+  default: false
+
+variables:
+- group: 'WebJobs SDK Testing'
+- name: buildNumber
+  value: $[ counter('constant', 12000) ]
```

Comment:
Is there any significance to "12000" being used here (instead of another constant value)?

Reply:
The AppVeyor build number is somewhere around 11990 at the moment, so 12000 is just to avoid overlap and keep an ascending sequence of build numbers.